framework/src/node/consensus/consensus.ts

```diff
@@ -300,6 +290,15 @@ export class Consensus {
 		await this._mutex.acquire();
 	}
 
+	// eslint-disable-next-line @typescript-eslint/require-await
+	public async isSynced(
```

Comment:
why this func always returns true? and whats the difference btwn `syncing` and `isSynced`?

Reply:
`syncing` is checking if it's currently running syncing or not.
`isSynced` is checking with given input, the chain is synced or not. it should be https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/node/forger/forger.ts#L93-L105 (will update)